src/style.css

```diff
@@ -1 +1,109 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url('./fonts/Avenir-Book.ttf') format('truetype');
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 600;
+  src: url('./fonts/Avenir-Heavy.ttf') format('truetype');
+}
+
+body {
+  margin: 0;
+  font-family: Avenir, Arial, sans-serif;
```

Comment:
According to the checklist, you should provide a fallback font. Here, you have Arial as a fallback for Avenir, which is good, but it's recommended to include a generic family name as well, like 'sans-serif'.